workers/approov-token-binding/README.md

```diff
@@ -0,0 +1,174 @@
+# Approov Token Binding Quick Start
+
+This quick start is for developers familiar with Cloudflare who are looking for a quick intro into how they can add [Approov](https://approov.io) into an existing project. Therefore this will guide you through the necessary steps for adding Approov to an existing site being routed through Cloudflare.
+
+We strongly advise you to try first this quick start in a staging environment, and just follow it in a production environment when you understand how Approov works within your Cloudflare deployment. So after you finish testing the Approov integration in your staging environment, just replace all occurrences of `staging` with `production` to deploy the Approov Cloudflare worker into your CLoudflare production infrastructure in order to protect your API with Approov.
+
+
+## Why?
+
+To lock down your API server to you mobile app, and you can read more about in the [README](/README.md#why) at the root of this repo.
+
+
+## How it works?
+
+See the overview in the [README](/README.md#how-it-works) at the root of this repo, or just dive into the [jwt-verifier.js](/workers/approov-token-binding/jwt-verifier.js) file, and take a look into the function `handleRequest` and `handleApproovTokenBindingCheck` to see how simple and easy the Approov Token check and the Approov Token Binding check are performed.
+
+
+## Requirements
+
+In order to to complete this quick start we will require the Approov and CLoudflare CLI tools to be installed in your system.
+
+### Cloudflare CLI Tool
+
+If not already installed in your system then you can follow the official docs [install instructions](https://developers.cloudflare.com/workers/tooling/wrangler/install/) for wrangler.
+
+### Approov CLI Tool
+
+If you haven't done it already, please follow [these instructions](https://approov.io/docs/latest/approov-installation/#approov-tool) from the Approov docs to download and install the [Approov CLI Tool](https://approov.io/docs/latest/approov-cli-tool-reference/).
+
+Don't forget to export your Approov developer management token:
+
+```text
+export APPROOV_MANAGEMENT_TOKEN=$(cat /path/to/development.token)
+```
+
+> **NOTE:**
+>
+> The above export is only valid for the current shell session.
+> If you open another shell you need to repeat it.
+> Add it to your `.bashrc` file in order to persist it across shell sessions.
+
+
+## Approov Setup
+
+In order to use Approov with Cloudflare we need to tell Approv what is the API domain being protected. We also need to set in Cloudflare the Approov Base64 encoded secret that will be used to verify the Approov tokens.
+
+### API Domain
+
+Approov needs to know the domain name for the API its authorized to issue valid tokens.
+
+Add it with:
+
+```text
+approov api -add your.api.domain.com
+```
+
+Now that Approov knows the domain for your API you get [dynamic certificate pinning](https://approov.io/docs/latest/approov-usage-documentation/#approov-dynamic-pinning) out of the box for free.
```

Comment:
Maybe state that pin is leaf public key by default, so must be the same across geographies

Reply:
Ok, I will ad it.